Tests/BaseRunnerTest.php

```diff
@@ -46,16 +48,12 @@ protected function setUp(): void
         parent::setUp();
         putenv('AWS_ACCESS_KEY_ID=' . AWS_ECR_ACCESS_KEY_ID);
         putenv('AWS_SECRET_ACCESS_KEY=' . AWS_ECR_SECRET_ACCESS_KEY);
-        $this->encryptorFactory = new ObjectEncryptorFactory(
+        $this->encryptor = ObjectEncryptorFactory::getAwsEncryptor(
```

Comment:
nemohl by tu být ten univerzální? nevidím důvod proč sem drátovat AWS

Reply:
mohl, ale v puvodni verzi se tu predavala konfigurace jen AWS (+ legacy cyphers), takze mi prislo naopak zbytecny to delat "univerzalne", protoze bych tam stejne hodil jen parametry pro AWS

(stejne tak i u vsech niz oznacenych vyskytu)

Reply:
no jo, ale to není proto, že by tam bylo potřeba testovat AWS, ale protože ten test je tak starej, že ještě žádnej azure nebyl. Mě přijde ujetý tam zadrátovat AWS, protože pak logicky očekávám, že je to specifický pro AWS a někde bude jinej test pro Azure. Tím spíš - že se tam někde dokonce předával jen dummy klíč